pyxmex/joiner.py

```diff
@@ -0,0 +1,17 @@
+class Joiner():
+    @classmethod
```

Comment:
What is the purpose of the @classmethod annotation? It seems like this function is being used like a static method, so why not the @staticmethod annotation?